core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -437,6 +487,47 @@ private ExprValue exprAddDateDays(ExprValue date, ExprValue days) {
         : exprValue);
   }
 
+  /**
+   * CONVERT_TZ function implementation for ExprValue.
+   * Returns null for time zones outside of +13:00 and -12:00.
```

Comment:
Your `isTimeZoneValid` function accepts TZ in range `[-14;+14]`. What MySQL accepts?

Reply:
Yes, it matches MySQL.